generative_models/VanillaGAN_TensorFlow/README.md

```diff
@@ -0,0 +1,103 @@
+# TensorFlow Implementation of VanillaGAN on MNIST Dataset
+
+### Usage
+```bash
+$ python3 main.py --epochs 50 --batch_size 128 --outdir 'desired location in system' 
+```
+NOTE: on Colab Notebook use following command:
+```bash
+!git clone link-to-repo
+%run main.py --epochs 50 --batch_size 128 --outdir 'desired location in system'
+```
+
```

Comment:
add help log, copy the output of `python3 main.py --help`

Reply:
done